CONTRIBUTIONS.md

```diff
@@ -0,0 +1,21 @@
+# Contributions
+
+## If you are interested in contributing, here are some ground rules:
+* Ideally, [talk to us](https://forum.unity.com/forums/input-system.103/) before doing the work -- we appreciate contributions, but we might already be working on the same thing, or we might have different opinions on how it should be implemented.
+
+## All contributions are subject to the [Unity Contribution Agreement(UCA)](https://unity3d.com/legal/licenses/Unity_Contribution_Agreement)
+By making a pull request, you are confirming agreement to the terms and conditions of the UCA, including that your Contributions are your original creation and that you have complete right and authority to make your Contributions.
+
+## Once you have a change ready following these ground rules. Simply make a pull request in Github
+
+Some notes:
+
+* We can rarely take a pull request directly and as is. The reason for this is that our CI will not be run on PRs coming from forks. So, in most cases we will have to incorporate your changes into a branch inside the repo and PR it into the `develop` branch via a separate PR. When we do so, we will clearly state that in your PR.

```

Comment:
I think that part does matter. Making a pull request, I'd expect someone to either take it or reject it. Whereas in our current workflow, even if we do take it, it will end up getting closed and not merged and your contribution won't really show as such in the history.

Reply:
Pushed